src/layer/AbstractLayer.ts

```diff
@@ -75,6 +78,7 @@ export class AbstractLayer {
     toTime: Date,
     maxFindTilesRequests: number = 50,
     tilesPerRequest: number = 50,
+    reqConfig?: RequestConfiguration,
```

Comment:
We have missed `findFlyovers` from the README, and the default values are `50` & `50` instead of `null` (as with `findTiles`). But we can leave this for a future cleanup if you wish. Fftr.